app/controllers/users_controller.rb

```diff
@@ -0,0 +1,29 @@
+class UsersController < ApplicationController
+  def show
+    @user = current_user
```

Comment:
DRY (do not repeat yourself), why you need @user variable, you have current_user helper already.